src/main/java/frc/robot/subsystems/Shoulder.java

```diff
@@ -76,7 +76,9 @@ public void periodic() {
     }
 
     private void useOutput(double output) {
-        double power = MathUtil.clamp(output, -ShoulderConstants.kMaxPower, ShoulderConstants.kMaxPower);
+        double power = output;
+        if (!this.getSetpoint().equals(ShoulderSetpoints.Climb))
+            power = MathUtil.clamp(output, -ShoulderConstants.kMaxPower, ShoulderConstants.kMaxPower);
         rightShoulderMotor.set(power);
```

Comment:
```suggestion
	if (this.getSetpoint().equals(ShoulderSetpoints.Climb)) {
	    power = power = MathUtil.clamp(output, -1.0, 1.0);
	} else {
            power = MathUtil.clamp(output, -ShoulderConstants.kMaxPower, ShoulderConstants.kMaxPower);
        }
        rightShoulderMotor.set(power);
```
You still need to clamp here to prevent a value greater than 1